modules/stanford_paragraph_postcard/config/install/core.entity_form_display.paragraph.stanford_postcard.default.yml

```diff
@@ -19,29 +19,35 @@ bundle: stanford_postcard
 mode: default
 content:
   field_p_postcard_body:
-    weight: 2
+    weight: 3
     settings:
       rows: 5
       placeholder: ''
     third_party_settings: {  }
     type: text_textarea
     region: content
   field_p_postcard_image:
-    weight: 1
+    weight: 2
     settings:
       progress_indicator: throbber
       preview_image_style: thumbnail
     third_party_settings: {  }
     type: image_image
     region: content
   field_p_postcard_more:
-    weight: 3
+    weight: 4
     settings:
       placeholder_url: ''
       placeholder_title: ''
     third_party_settings: {  }
     type: link_default
     region: content
+  field_p_postcard_orientation:
```

Comment:
It looks like this field got back in to the default display. As this field is to adjust display and not to actually be read by the end user does I think it makes sense to put it back to hidden. 

Also, the postcard component is ready. How about you attach it without the variants for now.